### ISSUE TITLE
[BUG]: Wrong imports defined in `tests/index.spec.ts`

### ISSUE THREAD

**Issue by JUNNNI** [snyk-snippets/modern-npm-package#9]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the bug

First of all, thank you very much for that repo, it's very useful.

I noticed that some of the imports located in the `index.spec.ts` are wrong.
See https://github.com/snyk-labs/modern-npm-package/blob/main/tests/index.spec.ts#L5

Indeed, I guess it should be 
```ts
import { helloWorld, goodBye } from '../src';
import commonPackage from '../src';

// instead of 

import { helloWorld, goodBye } from '../index';
import npmPackage from '../index';
```

### Steps To Reproduce

- Follow the installation guidelines from the README.md
- Run `npm test`
- Will results into the following error `tests/index.spec.ts:5:27 - error TS2307: Cannot find module '../index' or its corresponding type declarations.`

**Comment by leonardo-czanoch** [comment]
You probably have figure it out already, it should be:
```
import { helloWorld, goodBye } from '../src/index';
import npmPackage from '../src/index';

// instead of

import { helloWorld, goodBye } from '../index';
import npmPackage from '../index';
```

The owner has forsaken the newbies like us.

**Comment by clarkio** [comment]
This has been fixed now in #7. Thank you all for bringing this to our attention 👍
